frontend/static/css/snek.css

```diff
@@ -53,10 +52,12 @@ h1 {
 
 .snektion--u {
   align-self: flex-end;
+  transition: 2s all ease-in;
 }
 
 .snektion--n {
   align-self: flex-start;
+  transition: 2s all ease-out;
```

Comment:
So the CSS should all be in this file, is that right @oheydrew ?

Reply:
I've moved it there now

Reply:
Correct - Glad you saw that and fixed it up. I had commented on this but forgot to hit "submit review", I think :( haha